src/etc/check_missing_items.py

```diff
@@ -4,6 +4,8 @@
 # `index` or `paths`. It DOES NOT check that the structure of the produced json is actually in
 # any way correct, for example an empty map would pass.
 
+# FIXME: Better error output
```

Comment:
Why did you move the file?

Reply:
Other test helpers were in this folder, it seems the more 'standard' place. I can move it back or split it to a different PR, as it's somewhat unrelated.

Reply:
Hmm, given that `htmldocck` is here it seems fine to keep check_missing_items here too.